chasm/lib/activity/statemachine.go

```diff
@@ -120,6 +136,14 @@ var TransitionRescheduled = chasm.NewTransition(
 				Attempt: attempt.GetCount(),
 			})
 
+		a.recordOnAttemptedMetrics(
+			attempt.GetStartedTime().AsTime(),
+			event.namespace.String(),
+			event.handler,
+			event.breakdownMetricsByTaskQueue,
+			event.operationTag,
+			event.timeoutType)
```

Comment:
I'm finding it odd that we're referencing `GetStartedTime` time in a transition to `Scheduled`.